packages/coinstac-ui/app/render/state/ducks/maps.js

```diff
@@ -30,15 +34,33 @@ const INITIAL_STATE = {
 //   {type: 'boolean', name: 'isControl'},
 //   {type: 'number', name: 'age'}
 // ]
+//
+const getAllFiles = ((dirPath, arrayOfFiles) => {
+  const files = fs.readdirSync(dirPath)
+
+  arrayOfFiles = arrayOfFiles || [];
+
+  files.forEach((file) => {
+    if (fs.statSync(`${dirPath}/${file}`).isDirectory()) {
+      arrayOfFiles.push(path.join(__dirname, dirPath, '/', file));
+      arrayOfFiles = getAllFiles(`${dirPath}/${file}`, arrayOfFiles);
+    } else if (basename(file) !== '.DS_Store') {
+      arrayOfFiles.push(path.join(__dirname, dirPath, '/', file));
```

Comment:
I dont know if its such a bit deal copying these in to the comp, as the list of system files to ignore is probably not worth keeping, but doesnt matter either way

Reply:
Ok sounds good. Removing check for .DS_Store